src/main/java/frc/robot/RobotContainer.java

```diff
@@ -76,6 +84,12 @@ public RobotContainer() {
   private void configureBindings() {
     drivetrain.setDefaultCommand(new Drive(drivetrain, ()-> joyleft.getY(), ()-> joyleft.getX(), ()-> joyright.getX()));
   }
+  public void putShuffleboardCommands() {
+    controller.button(XboxController.Button.kA.value).onTrue(new RampMove(ramp, 400));
+    controller.button(XboxController.Button.kB.value).onTrue(new RampMove(ramp, 500));
+    SmartShuffleboard.putCommand("Ramp", "SetArmPID", new RampMove(ramp, 1200));
```

Comment:
If the command on the shuffleboard is working, convert this to be two commands on the shuffleboard: one that moves to 400, one that moves to 500, and remove the button mapping. This will be more scalable as more subsystems are added.